src/ResourceManager/Security/Commands.Security/Commands.Security.csproj

```diff
@@ -0,0 +1,177 @@
+    <Compile Include="Models\JitNetworkAccessPolicies\PSSecurityJitNetworkAccessPolicyVirtualMachinePort.cs" />
+    <Compile Include="Models\JitNetworkAccessPolicies\PSSecurityJitNetworkAccessRequestVirtualMachine.cs" />
+    <Compile Include="Models\JitNetworkAccessPolicies\PSSecurityJitNetworkAccessPolicyRequest.cs" />
+    <Compile Include="Models\JitNetworkAccessPolicies\PSSecurityJitNetworkAccessPolicy.cs" />
+    <Compile Include="Models\JitNetworkAccessPolicies\PSSecurityJitNetworkAccessPolicyConverters.cs" />
+    <Compile Include="Models\ExternalSecuritySolutions\PSSecurityAtaExternalSecuritySolution.cs" />
+    <Compile Include="Models\ExternalSecuritySolutions\PSSecurityCefExternalSecuritySolution.cs" />
+    <Compile Include="Models\ExternalSecuritySolutions\PSSecurityAadExternalSecuritySolution.cs" />
+    <Compile Include="Models\ExternalSecuritySolutions\PSSecurityExternalSecuritySolution.cs" />
+    <Compile Include="Models\ExternalSecuritySolutions\PSSecurityExternalSecuritySolutionConverters.cs" />
+    <Compile Include="Models\DiscoveredSecuritySolutions\PSSecurityDiscoveredSecuritySolution.cs" />
+    <Compile Include="Models\DiscoveredSecuritySolutions\PSSecurityDiscoveredSecuritySolutionConverters.cs" />
+    <Compile Include="Models\Compliances\PSSecurityComplianceSegment.cs" />
+    <Compile Include="Models\Compliances\PSSecurityCompliance.cs" />
+    <Compile Include="Models\Compliances\PSSecurityComplianceConverters.cs" />
+    <Compile Include="Models\AutoProvisioningSettings\PSSecurityAutoProvisioningSetting.cs" />
+    <Compile Include="Models\AutoProvisioningSettings\PSSecurityAutoProvisioningSettingConverters.cs" />
+    <Compile Include="Models\JitNetworkAccessPolicies\PSSecurityJitNetworkAccessPortRule.cs" />
+    <Compile Include="Models\Locations\PSSecurityLocation.cs" />
+    <Compile Include="Models\Locations\PSSecurityLocationConverters.cs" />
+    <Compile Include="Models\Pricings\PSSecurityPricing.cs" />
+    <Compile Include="Models\Pricings\PSSecurityPricingConverters.cs" />
+    <Compile Include="Models\SecurityContact\PSSecurityContact.cs" />
+    <Compile Include="Models\SecurityContact\PSSecurityContactConverters.cs" />
+    <Compile Include="Models\Tasks\PSSecurityTaskConverters.cs" />
+    <Compile Include="Models\WorkspaceSettings\PSSecurityWorkspaceSetting.cs" />
+    <Compile Include="Models\WorkspaceSettings\PSSecurityWorkspaceSettingConverters.cs" />
+    <Compile Include="SecurityCenterCmdletBase.cs" />
+    <Compile Include="Models\Tasks\PSSecurityTask.cs" />
+    <Compile Include="Models\Tasks\PSTaskSeverity.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Include="AzureRM.Security.psd1">
+      <Link>AzureRM.Security.psd1</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="AzureRM.Security.psd1" />
+    <None Include="help\AzureRM.Security.md" />
```

Comment:
@chlahav instead of including each individual `.md` file here, you can change it to the following:

```
<None Incude="help\*.md" />
```

This will prevent you from having to manually include any new markdown files moving forward.

Reply:
Done